modules/swagger-core/src/main/java/io/swagger/util/ParameterProcessor.java

```diff
@@ -144,6 +149,44 @@ public static Parameter applyAnnotations(OpenAPI openAPI, Parameter parameter, T
         return parameter;
     }
 
+    public static void setParameterExplode(Parameter parameter, io.swagger.oas.annotations.Parameter p) {
+        if (isExplodable(p)) {
+            if (Explode.TRUE.equals(p.explode())) {
+                parameter.setExplode(Boolean.TRUE);
+            } else if (Explode.FALSE.equals(p.explode())) {
+                parameter.setExplode(Boolean.FALSE);
+            }
+        }
+    }
+
+    private static boolean isExplodable(io.swagger.oas.annotations.Parameter p) {
+        io.swagger.oas.annotations.media.Schema schema = p.schema();
+        boolean explode = true;
+        if (schema != null) {
+            Class implementation = schema.implementation();
+            if (implementation == Void.class) {
+                if (!schema.type().equals("object") && !schema.type().equals("array")) {
+                    explode = false;
+                }
+            }
+        }
+        return explode;
+    }
+
+    public static void setParameterStyle(Parameter parameter, io.swagger.oas.annotations.Parameter p) {
+        if (isExplodable(p)) {
+            if (StringUtils.isNotBlank(p.style())) {
+                parameter.setStyle(Parameter.StyleEnum.valueOf(p.style()));
```

Comment:
Done! The Enums are now in lower case: mapper.configure(SerializationFeature.WRITE_ENUMS_USING_TO_STRING, true);